server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -602,6 +609,67 @@ public void vaultExportShouldResponseIdempotentSideEffect() throws Exception {
         }
     }
 
+    @Test
+    public void vaultPurgeShouldDeleteExpiredMessageThenExportWithNoEntry() throws Exception {
```

Comment:
`vaultPurgeShouldMakeExportProduceEmptyZipWhenAllMessagesAreExpired` take it if you think it's better